build.sbt

```diff
@@ -23,6 +23,7 @@ val postgresVersion            = "42.5.1"
 val skunkVersion               = "0.3.2"
 val lucumaSsoVersion           = "0.4.4"
 val testcontainersScalaVersion = "0.40.12"
+val paigesVersion              = "0.4.2"
```

Comment:
This is for the GraphQL query pretty-printer, which we should upstream to Grackle once we're sure we like it. I added code to the main mapping to print the query out on each request.